lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+<div class="starbook">
+  <h1>StarBook <small>Mk V</small></h1>
+  <p>with elementary OS</p>
+  <div>
+    <a href="https://en.starlabs.systems/products/starbook/elementary?rfsn=4227837.e8f025" class="button suggested">Buy Now</a>
+  </div>
+
+  <section id="stealth">
+    <img alt="StarBook Mk V, partially closed, from top" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/stealth.png") %>" />
+    <h3>Stealth power.</h3>
+    <p>StarBook Mk V looks unassuming in matte black anodized aluminum, but it’s engineered to pack a punch. 11th-gen Intel Core processors deliver performance on demand while sipping power. Low-latency memory keeps everything snappy. And the over-provisioned solid state StarDrive offers incredible speeds for opening your apps and files.</p>
```

Comment:
Could we split this into columns or bullets? I find with information in paragraphs like this I kinda end up glossing over everything but the headings:

# Stealth power.
Unassuming matte black anodized aluminum, but engineered to pack a punch.

|      |     |     |
| --- |--- | --- |
| **11th-gen Intel Core processors**  | **Low-latency memory** | **Over-provisioned solid state StarDrive**  |
| Performance on demand while sipping power | keeps everything snappy | Incredible speeds for opening apps and files


Is there something more definitive we can say about these components? Like for the GPU is there some kind of comparison we can make? Is 11th-gen the latest? What is low latency memory? What does "over-provisioned" mean? What makes the StarDrive significantly different from a regular SSD that it has a brand name?

We don't mention graphics at all here. Should we?